### ISSUE TITLE
Hunter Skilltree

### ISSUE THREAD

**Issue by Radschpud** [dalaranwow/dalaran-wow#4848]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. I am BM Hunter lvl 45. I try to set a skillpoint in my skill tree. (improved revive pet 2/2) but I cant. I reskilled the tree some days ago and just put 1 point in improved revive pet because I wantet bestial wrath first. now I want to set the other point.
2. it shoudn be a problem to set points anywhere in open lines.
3. I dont know. I dont want to play another char up. and I dont want to spent unnessesary money ;-)
![wowscrnshot_081817_210914](https://user-images.githubusercontent.com/14928789/29473938-f356e2fc-8459-11e7-8251-1b0d15ecbab9.jpg)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)

This issue can be solved by a gamemaster who will inspect you talents and grant it to you if missing. Thank you.

It's not a general problem, but instead a individual problem for you character.
